processor/resourcedetectionprocessor/README.md

```diff
@@ -0,0 +1,29 @@
+# Resource Detection Processor
+
+Supported pipeline types: metrics, traces
+
+The resource detection processor can be used to detect resource information from the host,
+in a format that conforms to the OpenTelemetry resource semantic conventions, and append or
+override the resource value in traces and metrics with this information.
+
+Currently supported detectors include:
+
+* Environment Variable: Reads resource information from the `OTEL_RESOURCE` environment
+variable. This is expected to be in the format `<key1>=<value1>,<key2>=<value2>,...` as
+defined in the OpenTelemetry resource specification.
```

Comment:
Yes, although this doesn't actually exist yet 😄 

I will add a link after https://github.com/open-telemetry/oteps/pull/111 is merged, and the spec gets updated

Reply:
Do you think that will get merged by the time we want to merge this? If not maybe we should indicate the speculative nature of it and that it may change?

Reply:
Yea, good point. Updated.